src/Components/benchmarkapps/BlazingPizza.Server/NuGet.config

```diff
@@ -2,11 +2,6 @@
 <configuration>
     <packageSources>
         <clear />
-        <add key="dotnet-core" value="https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json" />
-        <add key="extensions" value="https://dotnetfeed.blob.core.windows.net/aspnet-extensions/index.json" />
-        <add key="entityframeworkcore" value="https://dotnetfeed.blob.core.windows.net/aspnet-entityframeworkcore/index.json" />
-        <add key="aspnetcore-tooling" value="https://dotnetfeed.blob.core.windows.net/aspnet-aspnetcore-tooling/index.json" />
-        <add key="aspnetcore" value="https://dotnetfeed.blob.core.windows.net/aspnet-aspnetcore/index.json" />
         <add key="NuGet.org" value="https://api.nuget.org/v3/index.json" />
```

Comment:
@mmitche is this ok?

Reply:
Should be, all those feeds are defunct.

Reply:
> Should be, all those feeds are defunct.

I meant the presence of nuget.org

Reply:
Yes, because there is a <clear /> element and nothing but nuget.org